public/grpc-client.js

```diff
@@ -72,6 +73,11 @@ module.exports.init = async function({
   });
 
   ipcMain.on('lndInit', event => {
+    const macaroonCreds = getMacaroonCreds(lndSettingsDir, network);
+    credentials = grpc.credentials.combineChannelCredentials(
+      credentials,
+      macaroonCreds
+    );
```

Comment:
So the locker doesn't need macaroons? Only the `lnrpc.Lightning` service?

Reply:
If the wallet hasn't been initialized, the macaroons don't exist yet. The wallet creation process involves creating the macaroons :)